master/docs/manual/cfg-reporters.rst

```diff
@@ -899,3 +899,41 @@ It uses private token auth, and the token owner is required to have at least rep
     :param string baseURL: the base url of the GitLab host, up to and optionally including the first `/` of the path. Do not include /api/
     :param string verbose: Be more verbose
 
+SyslogStatusPush
```

Comment:
you will want to add `.. bb:reporter: SyslogStatusPush` before this header (this will include this particular reporter in the correct index and will allow to x-ref it in other places (should it be necessary))